Lesson4/homework4.py

```diff
@@ -0,0 +1,70 @@
+# Task 1
+print('Task 1: \"Removing letters\"')
+fruits = 'apple mango banana'
+my_str = input('Enter string: ')
+letters = input('Enter letter to delete: ')
+result1 = my_str.replace((letters), '')
+print(f'Enter string without specified letters: {result1}')
+
+# Task 2
+print('\nTask 2: \"Capital letters\"')
+text = 'This is some test STRING'
+my_text = input('Enter string: ')
+result2 = my_text.title()
+print(f'Enter string with capital letters: {result2}')
+
+# Task 3
+print('\nTask 3: \"Reverse strings\"')
+browser = 'Google Chrome'
+my_browser = input('Enter browser name: ')
+result3 = my_browser[::-1]
+print(f'Enter with reverse string : {result3}')
+
+# Task 4
+print('\nTask 4: \"Compare of strings\"')
+string_1 = 'abcdef12345'
+string_2 = 'London Paris'
+string__1 = input('Enter first string: ')
+string__2 = input('Enter second string: ')
+
+if len(string__1) == len(string__2) != 0:
+    print('The strings match')
+elif len(string__1) > len(string__2):
+    print('The first string is longer than the second')
+elif len(string__1) < len(string__2):
+    print('The first string is shorter than the second')
+else:
+    print('Both strings are empty')
+
```

Comment:
This task can be simplier
str1 = input("Enter first string: ")
str2 = input("Enter second string: ")
result = str1 == str2

print(f'String {str1} and string {str2} have equals result {result}')